docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -1,20 +1,18 @@
 Developer Setup
 ===============
 
-.. include:: links.rst
-
 Creating a ``bash`` development shell
 -------------------------------------
 
-.. include:: ../miniforge3_instructions.rst
+.. include:: ../../shared/miniforge3_instructions.rst
```

Comment:
I created a `docs/shared` directory for snippets, fragments, etc., that we might want to share between doc pages. We only have one such fragment now, and I've moved it there.